blockchain/storage/leveldb/leveldb.go

```diff
@@ -25,7 +31,13 @@ func Factory(config map[string]interface{}, logger hclog.Logger) (storage.Storag
 
 // NewLevelDBStorage creates the new storage reference with leveldb
 func NewLevelDBStorage(path string, logger hclog.Logger) (storage.Storage, error) {
-	db, err := leveldb.OpenFile(path, nil)
+	options := opt.Options{}
+	// Set default options
+	options.OpenFilesCacheCapacity = DefaultHandles
+	options.BlockCacheCapacity = DefaultCache / 2 * opt.MiB
+	options.WriteBuffer = DefaultCache / 4 * opt.MiB // Two of these are used internally
+
+	db, err := leveldb.OpenFile(path, &options)
```

Comment:
Nitpick:

```suggestion
	options := &opt.Options{}
	// Set default options
	options.OpenFilesCacheCapacity = DefaultHandles
	options.BlockCacheCapacity = DefaultCache / 2 * opt.MiB
	options.WriteBuffer = DefaultCache / 4 * opt.MiB // Two of these are used internally

	db, err := leveldb.OpenFile(path, options)
```

Reply:
Why don't we put initialization of fields in brackets? like:
```
options := &opt.Options{
  OpenFilesCacheCapacity: ...
  ...
}
```

Reply:
https://github.com/0xPolygon/polygon-edge/pull/1569/commits/5ae832c5e3697bd99a48d8345ce5825cccd9328a